assets/src/edit-story/components/panels/design/layer/layerPanel.js

```diff
@@ -27,31 +28,57 @@ import { LAYER_HEIGHT, DEFAULT_LAYERS_VISIBLE } from './constants';
 import LayerList from './layerList';
 import useLayers from './useLayers';
 
+const Container = styled.div`
+  margin: 0 4px;
+  filter: drop-shadow(0px -4px 8px rgba(0, 0, 0, 0.2));
+`;
+
+const StyledPanelTitle = styled(PanelTitle)`
+  height: 48px;
+  border-top-left-radius: ${({ theme }) => theme.borders.radius.small};
+  border-top-right-radius: ${({ theme }) => theme.borders.radius.small};
+`;
+
+const DividerContainer = styled.div`
+  background: ${({ theme }) => theme.colors.bg.tertiary};
+`;
+
+const Divider = styled.div`
+  border-bottom: 1px solid ${({ theme }) => theme.colors.divider.tertiary};
+  margin: 0 16.5px;
```

Comment:
Sam said we should increase the spacing around the divider because it looked like this when the scrollbar appeared:
<img src="https://user-images.githubusercontent.com/22185279/110134719-f25be680-7d8a-11eb-821b-cc2e2143226c.png" width="500px:" />

I can change it back too but it needed a little bit of space. Should we change it back to 8px?

Chatted with Sam about this here: https://xwp.slack.com/archives/C01ESBB8ZN1/p1614791411042500

Reply:
re the slack thread Samuel posted, @samitron7 's response was `Just adding more padding between them would work.`

Reply:
Fine with me! Though half-pixels always bug me 🤷‍♂️